Map.h

```diff
@@ -78,20 +78,17 @@ void andPixelAR(uint8_t angle, uint8_t dAngle, uint8_t startRadius, uint8_t endR
 
 void antialiasPixelAR(uint8_t angle, uint8_t dAngle, uint8_t startRadius, uint8_t endRadius, CRGB color)
 {
-  uint16_t amax = qadd8(angle, dAngle);
-  uint8_t amin = qsub8(angle, dAngle);
-
   for (uint16_t i = 0; i < NUM_LEDS; i++) {
     uint8_t o = i;
 
     uint8_t ao = angles[o];
 
-    uint8_t adiff = qsub8(max(ao, angle), min(ao, angle));
-    uint8_t fade = qmul8(adiff, 32);
+    uint8_t adiff = min(sub8(ao,angle), sub8(angle, ao));
+    uint8_t fade = map(adiff, 0, dAngle, 0, 255);
```

Comment:
I think map8 might be faster here, for mapping from one uint8_t range to another.

Reply:
unfortunately the FastLED map8 function yields not the expected result
_map8: map from one full-range 8-bit value into a narrower range of 8-bit values_
In this case the incoming value is limited (by dAngle) and needs to be mapped to a full range. I found no equivalent function in the FastLED math library (also the lerp function don't help, they all expect a full range input value, i.e. "fraction".)

Reply:
Ah, sorry, you're right: http://fastled.io/docs/3.1/group__lib8tion.html#gaf65357c7aeca9778e5a2f80a6bac2b5d

```
map8: map from one full-range 8-bit value into a narrower range of 8-bit values, possibly a range of hues.
```